docs/source/dataframe-create.rst

```diff
@@ -58,29 +58,39 @@ and not actively maintained; use at your own risk.
 
 .. code-block:: Python
 
-   >>> from castra import Castra
-   >>> c = Castra(path='/my/castra/file')
-   >>> df = c.to_dask()
+   >>> from castra import Castra # doctest: +SKIP
+   >>> c = Castra(path='/my/castra/file') # doctest: +SKIP
+   >>> df = c.to_dask() # doctest: +SKIP
 
 .. _Castra: http://github.com/blaze/castra
 
 From HDF5
 ----------
 
-`HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_ is a Hierarchical Data Format (HDF) designed to store and organize large amounts of data.  Similar to the `pandas I\/O API <http://pandas.pydata.org/pandas-docs/stable/io.html>`_,    `dask <(http://dask.pydata.org/en/latest/index.html>`_ can create a DataFrame directly from `HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_ datasets.
+`HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_ is a Hierarchical Data Format (HDF) designed to store and organize large amounts of data.  
+
+First, we create a random HDF5 dataset.
 
 .. code-block:: Python
 
-   >>> # Load hdf5 into dask DataFrame
-   >>> dd.read_hdf('myfile.1.hdf5', '/x', chunksize=1000000)
-   
-   
-   >>> # OR Load multiple hdf5 files into dask DataFrame
-   >>> dd.read_hdf('myfile.*.hdf5', '/x', chunksize=1000000)  
+   >>> import h5py
+   >>> import numpy as np
+   >>> f = h5py.File("myfile.hdf5", "w")
+   >>> dset = f.create_dataset("mydataset", (1000000,), dtype='i')
+   >>> dset[...] = np.arange(1000000)
+   >>> print dset.shape
+   (1000000,)
```

Comment:
The purpose of docstings is to be clear and simple, I think your original version with a +SKIP might be better.